src/test/java/seedu/address/logic/commands/ViewHistoryCommandTest.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.model.VersionedTutorsPet.INITIAL_COMMIT_MESSAGE;
+import static seedu.address.model.VersionedTutorsPetTest.COMMIT_MESSAGE_1;
+import static seedu.address.model.VersionedTutorsPetTest.COMMIT_MESSAGE_2;
+import static seedu.address.model.VersionedTutorsPetTest.COMMIT_MESSAGE_3;
+import static seedu.address.testutil.TypicalTutorsPet.getTypicalTutorsPet;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ViewHistoryCommandTest {
+
+    private Model model;
+    private Model expectedModel;
+
+    @BeforeEach
+    public void setUp() {
+        model = new ModelManager(getTypicalTutorsPet(), new UserPrefs());
+        model.commit(COMMIT_MESSAGE_1);
+        model.commit(COMMIT_MESSAGE_2);
+        model.commit(COMMIT_MESSAGE_3);
+        expectedModel = new ModelManager(getTypicalTutorsPet(), new UserPrefs());
+        expectedModel.commit(COMMIT_MESSAGE_1);
+        expectedModel.commit(COMMIT_MESSAGE_2);
+        expectedModel.commit(COMMIT_MESSAGE_3);
+    }
+
+    @Test
+    public void execute_latestState_success() {
+        String expectedMessage = String.format(ViewHistoryCommand.MESSAGE_TEMPLATE,
+                "\n" + ViewHistoryCommand.CURRENT_INDICATOR + COMMIT_MESSAGE_3 + "\n"
+                + COMMIT_MESSAGE_2 + "\n"
+                + COMMIT_MESSAGE_1 + "\n"
+                + INITIAL_COMMIT_MESSAGE);
```

Comment:
Because of the need to include the `CURRENT_INDICATOR` for a particular entry, which is rather specific to `ViewHistoryCommand`, the method in `StringUtil` would be oddly specialised for a method in the `commons` package.

Instead, I think I would abstract out the creation of the message to a helper function that resides within the `ViewHistoryCommand` class.